src/content/docs/nrql/using-nrql/subquery-joins.mdx

```diff
@@ -63,11 +63,27 @@ FROM Event [INNER|LEFT] JOIN (subquery) ON [key =] key SELECT ...
 ```
 
 There are a few simple rules to the syntax:
+
 * The JOIN clause must always follow immediately after the FROM clause.
 * The JOIN can be prefixed with the join type. INNER or LEFT is optional, and defaults to INNER when omitted.
 * Parentheses containing a subquery must immediately follow the JOIN clause.
 * The ON clause must immediately follow the subquery and has two forms (more details below).
 
+You can also have multiple joins in one query, too. For example, this query uses two JOINS within subqueries:
```

Comment:
I don't think you need the "also" and the ",too" in the same sentence. Whichever one you like the most is the only one you need!

Reply:
LOL fair! fixing~